packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/splitter.markup.tests.js

```diff
@@ -28,27 +35,143 @@ const moduleConfig = {
 
             init(options);
         };
+
+        this.getItems = () => {
+            return this.$element.find(`.${SPLITTER_ITEM_CLASS}`);
+        };
+        this.getNestedSplitter = () => {
+            return this.$element.find(`.${SPLITTER_CLASS}`);
+        };
+
     }
 };
 
-QUnit.module('Splitter markup', moduleConfig, () => {
-    QUnit.test('Splitter should have dx-splitter class', function(assert) {
+QUnit.module('Render', moduleConfig, () => {
+    QUnit.test('with single pane', function(assert) {
```

Comment:
we have similar tests in splitter.tests ( splitter items count), lets remove tests from splitter.tests then

Reply:
I suggest to stay this simple configuration. Yes, we have similar setups in other tests, but those tests check more complicated situations.